src/test/java/com/wikia/webdriver/testcases/discussions/PostDetailsSharing.java

```diff
@@ -0,0 +1,63 @@
+package com.wikia.webdriver.testcases.discussions;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.discussions.PostDetailsPage;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Content X-Wing Wikia
+ */
+@Test(groups = {"Discussions", "PostDetailsSharing"})
+public class PostDetailsSharing extends NewTestTemplate {
+
+  private static final String DESKTOP_RESOLUTION = "1366x768";
+  private static final String MOBILE_RESOLUTION = "600x800";
+
+  private Credentials credentials = Configuration.getCredentials();
+
+  /**
+   * ANONYMOUS USER
+   */
+
+  @Test
+  @Execute(asUser = User.ANONYMOUS)
+  @InBrowser(browserSize = DESKTOP_RESOLUTION)
+  public void socialNetworkIconsAreDisplayedToAnonymousUser() {
+    socialNetworkIconsAreDisplayed(new String[]{"Facebook", "Twitter", "Reddit", "Tumblr"});}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)